storage/persistence/spanner/setup_test.go

```diff
@@ -25,71 +13,6 @@ var (
 	ctx      = context.Background()
 )
 
-func CreateInstance(ctx context.Context, cfg *config.SpannerConfig) error {
```

Comment:
hmmm... I prefer to keep CreateInstance/CreateDatabase in the spanner package since they are meanless when outside the spanner

Reply:
The reason is NewManager depends on spanner.  And according to the test order, manager_test is executed before spanner_test.

Reply:
I tried it first but it seemed not working since "spanner.CreateDatabase" is in a test file and cannot be exported.

Reply:
I see...

Reply:
I'll figure out if there is any other way of doing this.

Reply:
The only way should rename file name without `_test` suffix

Reply:
Personally speaking, I think these  manager tests are somehow redundant because persistence is nested in manager and that part of codes have already been covered in persistence unit tests.

Reply:
Yes, I also prefer to using mock storage if we really want to test the manager.

Reply:
Okay then, I will archive this PR for now.